README.md

```diff
@@ -351,6 +351,18 @@ ______________________________________________________________________
 
 ______________________________________________________________________
 
+## Lightning Lite
+
+<div align="center">
+  <img src="docs/source/_static/images/lightning_lite/lite.gif" max-height="250px">
```

Comment:
This seems a bit too big.

Reply:
@rohitgr7 Maybe we could remove the image (because it has nothing meaningful really) and add a minimal code snippet showing the diff between a normal PyTorch script and one with Lite to show how easy it is to scale their training.

Reply:
I'll reduce the size.

@akihironitta the banner is added to attract attention. The diff might be too long so I linked the blogpost for a better read.